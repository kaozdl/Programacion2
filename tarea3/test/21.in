# insertar_binario
insertar_binario .
insertar_binario (15,q) .
insertar_binario (14,c) .
insertar_binario (16,d) (13,t) (-3,m) .
Fin
