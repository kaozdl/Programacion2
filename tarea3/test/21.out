1>
2>
3>
(15,q)
0
0
4>
(15,q)
-(14,c)
1
0
5>
-(13,t)
(15,q)
---(-3,m)
--(16,d)
-(14,c)
3
1
6>Fin.
